R/commonAnovaBayesian.R

```diff
@@ -2380,7 +2443,42 @@ BANOVAcomputMatchedInclusion <- function(effectNames, effects.matrix, interactio
   } else {
     # put the null-model first
     i <- length(out)
-    return(c(out[[i]], out[-i]))
+    out <- c(out[[i]], out[-i])
+
+    # BayesFactor has the nasty habit of changing the order of interactions whenever one of the components is
```

Comment:
I don't like it either. In all honesty, one day we should roll our own solution for BayesFactor/ BAS. Both introduce some annoying issues and we'd be better off with something we can control (and fix within a reasonable timeframe). This would also make certain feature requests possible (different priors, etc.)